### ISSUE TITLE
Printing functions

### ISSUE THREAD

**Issue by thedavidchu** [thedavidchu/c_lib#11]
* Print arr and tbl to stdout

```c
/* Panic upon failure */
void arr_print(const arr *const restrict me, void (*item_print)(const void *const restrict));
void tbl_print(const tbl *const restrict me, void (*key_print)(const void *const restrict),
        void (*value_print)(const void *const restrict));
```

**Comment by thedavidchu** [comment]
After #3

**Comment by thedavidchu** [comment]
Fixed in #3.
